WebRoot/css/zm.css

```diff
@@ -4109,40 +4109,80 @@ HTML>BODY .appt-selected .appt_allday_body
 	vertical-align:middle;
 }
 
-.ZmFilterRuleDialog .FilterRuleSection,
-.ZmFilterRuleDialog .FilterActionSection {
-	background-color:rgba(255,255,255,.5);
-	border:1px solid rgba(0,0,0,.25);
-	margin:.25em 1em 1em;
-	padding:0px;
-#IFDEF MSIE
+.ZmFilterRuleDialog .WindowOuterContainer {
+	@FilterRuleDialogOuterContainer@
+}
+
+.ZmFilterRuleDialog .ZmFilterNameSection {
+	@FilterRuleDialogFilterNameSection@
+}
+
+.ZmFilterRuleDialog .ZmFilterName {
+	@FilterRuleDialogFilterName@
+}
+
+.ZmFilterRuleDialog .ZmFilterActiveLabel {
+	@FilterRuleDialogCheckBoxLabel@
+}
+
+.ZmFilterRuleDialog .ZmFilterNameInput {
+	@FilterRuleDialogInputWrapper@
+}
+
+.ZmFilterRuleDialog .ZmFilterRuleSection,
+.ZmFilterRuleDialog .ZmFilterActionSection {
+	@FilterRuleDialogFieldSet@
+	#IFDEF MSIE
 	/* IE sucks at rgba backgrounds, so using gradient hack */
 	filter:progid:DXImageTransform.Microsoft.gradient(GradientType=0,startColorstr='#80FFFFFF',endColorstr='#80FFFFFF');
 #ENDIF
 }
 
-.ZmFilterRuleDialog .FilterRuleSection>DIV,
-.ZmFilterRuleDialog .FilterActionSection>DIV {
-	padding:.5em 1.5em .5em .5em;
+.ZmFilterRuleDialog .ZmFilterRuleSectionContent,
+.ZmFilterRuleDialog .ZmFilterActionSectionContent {
+	@FilterRuleDialogFieldSetContent@
 }
 
-.FilterActions>TBODY>TR>TD {
-	padding-right:6px;
+.ZmFilterRuleDialog .ZmFilterAddConditionSection,
+.ZmFilterRuleDialog .ZmFilterAddActionSection {
+	@FilterRuleDialogAddSection@
+}
+
+.ZmFilterRuleDialog .ZmFilterAddConditionIcon,
+.ZmFilterRuleDialog  .ZmFilterAddActionIcon {
+	@FilterRuleDialogAddIconWrapper@
+}
+
+.ZmFilterRuleDialog .ZmFilterBrowse td {
+	@FilterRuleDialogBrowseButton@
+
+}
+
+.ZmFilterRuleDialog .DwtInputField INPUT[type='text'] {
+	@FilterRuleDialogInputField@
+}
+
+.ZmFilterRuleDialog input:focus {
+	@FilterRuleDialogInputFocused@
+}
+
+.ZmFilterRuleDialog .filterActions>tbody>tr> td:first-of-type {
+	@FilterRuleDialogActionSectionSelectWidget@
 }
 
 #IFDEF MSIE
 	/* IE won't pick up padding on the table cell elements unless the collapse is turned off */
-.FilterAddRemoveButtons {
-	border-collapse:separate;
+.ZmFilterRuleDialog .ZmFilterRemoveButtons {
```

Comment:
Do we need this rule only for IE? Have we verified that other browsers don't need it?